README.md

```diff
@@ -28,3 +28,7 @@ This scenario is published as part of the kube academy course ["Hands-On with Ku
 
 The training portal is configured to allow anonymous access. For your own
 workshop content you should consider removing anonymous access.
+
+## Review Environment
```

Comment:
I'm afraid that the url mentioned here is rather ephemeral.  It's very likely that next week that url would be broken.
If I were to update and redeploy the workshop on that particular cluster, educates may end up creating a new workshop environment whose name doesn't match "kube-academy-labs-w02."
The workshop could also be deployed as part of a training portal with a different name, in which case the url would change as well.

I'd like to keep these urls in our issue manager stories, but out of this readme file, because an educates workshop repository is just content, and could be deployed to many environments.

Reply:
Works for me to put them in the stories. I'll delete this PR.